packages/block-library/src/comments-pagination-next/index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Server-side rendering of the `core/comments-pagination-next` block.
+ *
+ * @package WordPress
+ */
+
+
+/**
+ * Applies the block attributes to the block markup without using a wrapper.
+ *
+ * @return string Returns the styles and classes defined on the block editor for the block.
+ */
+function add_next_comments_link_attributes() {
+	if ( ! function_exists( 'get_block_wrapper_attributes' ) ) {
```

Comment:
This function was added in WordPress 5.6 to there is no need to add the check:

https://github.com/WordPress/wordpress-develop/blob/77232140658c01121937318f0d0c1969d6827aec/src/wp-includes/class-wp-block-supports.php#L164-L169